pkg/handler/handler.go

```diff
@@ -174,14 +174,22 @@ func (h handler) deprovision(w http.ResponseWriter, r *http.Request, params map[
 		writeResponse(w, http.StatusBadRequest, broker.ErrorResponse{Description: "invalid instance_uuid"})
 		return
 	}
+
 	var async bool
 	queryparams := r.URL.Query()
```

Comment:
But only if they are coming in as a form. If they are json or coming in on the actual url hook. I'd be careful making that change blindly.